### DISCUSSION TITLE
Runoff is significantly different when Ks is present

### DISCUSSION THREAD

**Discussion by BrianCollinss** [APSIMInitiative/ApsimX#8975] [Q&A]
I downloaded some data using the World Modellers API. This API downloads Ks values as well. If I am not wrong, when Ks is present, it is used to estimate the infiltration rate and downward flux. However, when I leave Ks in, almost 70% of irrigation water is lost via runoff while it is much less when I remove Ks and let APSIM use SWCON. I would like to know if anyone else has experienced the same thing. I wonder if APSIM has been tested sufficiently with Ks.

**Comment by hut104** [comment]
The SCS Runoff approach used in the soilwater model allows soil water content to affect surface water infiltration via a calculation of the antecedant soil water contitions.  This is evaluated to a depth of 450mm, with weighting  toward moisture near the surface.

Ks is used to set a throttle for infiltration under saturated conditions, at any point within the profile.  

In some situations, there may be a throttle to infiltration due to soil constraints (e.g. high ESP or texture contrast).  Water will perch above the constraint.  If the perched water table reaches the surface, infiltration is affected and runoff will be enhanced.

Users should ensure that Ks values in layers and bare soil runoff curve number are in harmony.  The curve number should reflect the basic soil type.  Ks is usually high enough have little impact on daily runoff totals.  In some cases, Ks can be reduced by the users to account for constraints to infiltration somewhere in the profile. 

The approach and assumptions are also used in the SWIM model.

Testing of this has been undertaken in simulations of 
- long term monitoring of Cl movement (ie leaching) in fields with sub-soil constraints (ie  high ESP at depth)
- water balance in duplex soils in Australia and North America.

I am not aware of what evaluation has been undertaken WRT the Ks values in the World Modellers API (or Curve number).

Note that you may also be observing impacts of layer thickness in your simulations.
